src/Stomp/Parser.php

```diff
@@ -104,7 +111,7 @@ public function parse()
      */
     private function parseToFrame($source)
     {
-        list ($header, $body) = explode("\n\n", ltrim($source), 2);
+        list ($header, $body) = explode($this->delimiter, ltrim($source), 2);
```

Comment:
First time we reqch this linedelimiter will be null?

Reply:
That's true, but it's called by `parse`, after `getFrameEnd` returns.